tests/test_create_library_files.py

```diff
@@ -0,0 +1,130 @@
+import os
+import pytest
+import pandas as pd
+from .test_sqlite import check_sqlite_files_are_equal
+from ms2query.create_library_files import CreateFilesForLibrary
+from ms2query.app_helpers import load_pickled_file
+
+
```

Comment:
```suggestion

@pytest.fixture
def path_to_general_test_files():
    return os.path.join(
        os.path.split(os.path.dirname(__file__))[0],
        'tests/test_files/general_test_files')
```

Reply:
Idea here is to remove some redundant parts and make the test functions easier to read.

Reply:
Cool! Did not use that before, but is a bit cleaner indeed

Reply:
Also just learnt this recently :)